src/shared/ListRepo/ListRepo.jsx

```diff
@@ -74,6 +106,7 @@ function ListRepo({ canRefetch }) {
             sortItem={sortItem}
             owner={owner}
             searchValue={params.search}
+            mayIncludeDemo={true}
```

Comment:
nit: keys are implicitly true if we nix the ={true}

Reply:
good call! fixed!